src/main/groovy/com/avast/gradle/dockercompose/ComposeExecutor.groovy

```diff
@@ -19,7 +19,7 @@ class ComposeExecutor {
         this.logger = settings.project.logger
     }
 
-    private void executeWithCustomOutput(OutputStream os, Boolean ignoreExitValue, String... args) {
+    void executeWithCustomOutput(OutputStream os, Boolean ignoreExitValue, String... args) {
```

Comment:
Public methods with `Boolean` argument are IMHO smell - please keep this method private and ensure there are two public methods with different names.